### PR TITLE
Updating Replication Log for MS Marco Experiments for pyserini

### PR DESCRIPTION
**Pull request by mayankanand007**
Environment: Ubuntu 20.04 on Google Compute Engine (150 GB SSD, 4 vCPUs, 16 GB memory)

No issues encountered.

### CODE REVIEW

**Comment by MXueguang** [comment]
just to confirm, all the results in the replication are exactly the same as the numbers in documents?

**Comment by mayankanand007** [comment]
Yes all numbers were the same

File: docs/experiments-msmarco-doc.md [hunk 1]
```diff
@@ -139,3 +139,4 @@ We can see that Anserini's (tuned) BM25 baseline is already much better than the
 + Results reproduced by [@printfCalvin](https://github.com/printfCalvin) on 2021-04-12 (commit [`0801f7f`](https://github.com/castorini/pyserini/commit/0801f7fb15e249f2e67901a6523d6ce68c667207))
 + Results replicated by [@saileshnankani](https://github.com/saileshnankani) on 2021-04-26 (commit [`6d48609`](https://github.com/castorini/pyserini/commit/6d486094137a26c8a0a57652a06ab4d42d5bce32))
 + Results replicated by [@andrewyguo](https://github.com/andrewyguo) on 2021-04-30 (commit [`ecfed61`](https://github.com/castorini/pyserini/commit/ecfed61bfba065aa958848cff96ba9f22609aeb1))
++ Results replicated by [@mayankanand007](https://github.com/mayankanand007) on 2021-05-04
```
Comment (lintool) [inline docs/experiments-msmarco-doc.md:142]: Please not the commit id from which you did the reproduction.

btw, replicated -> reproduced - change above instances also.

**Review by lintool — CHANGES_REQUESTED** [review]

File: docs/experiments-msmarco-doc.md (hunk 1 above)
Comment (mayankanand007) [reply docs/experiments-msmarco-doc.md:142]: Does the commit ID refer to the last commit in the master branch in the repo before we cloned it and used it to reproduce results?

Comment (lintool) [reply docs/experiments-msmarco-doc.md:142]: yes

Comment (mayankanand007) [reply docs/experiments-msmarco-doc.md]: Fixed.

**Review by lintool — APPROVED** [review]

**Pull request merged**